services/gateway/apis/http-version3/methods/blockchainAppsMetadata.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
 	params: {
 		chainName: { optional: true, type: 'string', min: 1, max: 20, pattern: regex.NAME },
 		chainID: { optional: true, type: 'string', pattern: regex.CHAIN_ID_CSV },
+		isDefault: { optional: true, type: 'boolean', min: 1, pattern: regex.BOOLEAN },
```

Comment:
We do not really need a patter check for the `boolean` type. Instead, optionally, enforce the `convert` property to `false`, which also happens to be the default value.

Ref: https://github.com/icebob/fastest-validator#boolean

```suggestion
		isDefault: { optional: true, type: 'boolean', convert: false },
```